index.js

```diff
@@ -61,7 +104,8 @@ module.exports = function inspect_ (obj, opts, depth, seen) {
         for (var i = 0; i < obj.length; i++) {
             xs[i] = has(obj, i) ? inspect(obj[i], obj) : '';
         }
-        return '[ ' + xs.join(', ') + ' ]';
+        return listFmt('Array', false, xs,
+          { depth: depth, opts: opts, obj: obj });
```

Comment:
either function invocations should be entirely on one line, or, `(`, each argument, and `)` should be on separate lines.